os/arch/arm/src/s5j/s5j_dma.h

```diff
@@ -51,14 +51,14 @@
 #define DMAC_FSRC                    (0x034)
 #define DMAC_FTRD                    (0x038)
 
-#define DMAC_CH_FTR(n)               0x040 + 0x04 * (n)
-#define DMAC_CH_CSR(n)               (u32)(0x100 + 0x08 * (n))
-#define DMAC_CH_CPC(n)               0x104 + 0x08 * (n)
-#define DMAC_CH_SAR(n)               0x400 + 0x20 * (n)
-#define DMAC_CH_DAR(n)               0x404 + 0x20 * (n)
-#define DMAC_CH_CCR(n)               0x408 + 0x20 * (n)
-#define DMAC_CH_LC0(n)               0x40C + 0x20 * (n)
-#define DMAC_CH_LC1(n)               0x410 + 0x20 * (n)
+#define DMAC_CH_FTR(n)               0x040 + 0x04*(n)
```

Comment:
Why do you change this block? You make a problem

Reply:
Restored